float64[7] actual
float64[7] input
float64[7] friction_i
float64[7] tau_mis
float64[7] q_f
float64[7] dq_f
float64[7] ddq_f
float64[6] r_error
float64[6] s_out
float64[6] z_out
float64[6] F_e
float64[3] r_d
float64[6] impe_error
float64[3] r_out
float64[6] F_d
float64 Kdz
float64 Kfz
float64 Ddz
float64 time_data
float64[4] traj_


